source/extensions/filters/http/jwt_authn/jwks_cache.h

```diff
@@ -55,6 +56,9 @@ class JwksCache {
     // Check if a list of audiences are allowed.
     virtual bool areAudiencesAllowed(const std::vector<std::string>& audiences) const PURE;
 
+    // Check if a subject is allowed.
+    virtual bool isSubjectAllowed(const absl::string_view sub) const PURE;
```

Comment:
```suggestion
    virtual bool isSubjectAllowed(absl::string_view sub) const PURE;
```

nit: no top-level const in declaration.